### PR TITLE
Improve code readability and update requirements

### PR DESCRIPTION
**Pull request by joaomedeiros95**
- Update Redis version to 3.3.11 (the last one);
- Correct the max line size using pep8 convention;
- Add a new parameter to attempt that is the timestamp if the user wants to fix the timestamp between attempts

### CODE REVIEW

File: pylimit/tests/test_pylimit.py [hunk 1]
```diff
@@ -7,7 +7,8 @@
 class TestPyLimit(unittest.TestCase):
     def test_exception(self):
         limit = PyRateLimit()
```
Comment (biplap-sarkar) [inline pylimit/tests/test_pylimit.py]: Thanks for the pull request. This pull request is also introducing a breaking change where constructor without arguments is not valid anymore. As a result, tests are not working anymore. Will it be possible to update constructor arguments to that tests don't fail?

**Review by biplap-sarkar — CHANGES_REQUESTED** [review]

**Comment by biplap-sarkar** [comment]
Will it be possible to update the readme as well? The current example won't be valid anymore

**Comment by joaomedeiros95** [comment]
Thanks for the feedback @biplap-sarkar, all of the changes have been made.

**Review by biplap-sarkar — APPROVED** [review]

**Comment by rootux** [comment]
@biplap-sarkar This requires a new release as now the README states that you initialize using PyRateLimit(period=1,limit=10) but in fact the latest release is not updated with the README

**Pull request merged**